test/test_endpoints.py

```diff
@@ -154,13 +157,26 @@ def get_block_by_number(req_dict: dict):
     )
 
 
+def get_block_by_hash(req_dict: dict):
```

Comment:
Why should this function accepts a dict instead of a hash?

Reply:
Fixed in 353fa8954ff6b4dc95f5d23657f65882acfefcd0